### ISSUE TITLE
TYPO3 13

### ISSUE THREAD

**Issue by msollmann** [cdaecke/md_fullcalendar#27]
Hi,

are there any plans for TYPO3 13?

Regards
Michael

**Comment by cdaecke** [comment]
Yes, compatibility for TYPO3 v13 is planned, but there is no date yet.

Every contribution welcome!

**Comment by cdaecke** [comment]
Pull request https://github.com/cdaecke/md_fullcalendar/pull/29 adds support for TYPO3 v13.

Feedback and testers are welcome!

**Comment by cdaecke** [comment]
Version [5.0.0](https://github.com/cdaecke/md_fullcalendar/releases/tag/5.0.0) is released.

Thanks to @oliverschloebe for his contribution!
